chapters/fa/chapter4/1.mdx

```diff
@@ -0,0 +1,23 @@
+<div dir="rtl">
+# هاب هاگینگ‌فیس
+
+[هاب هاگینگ‌فیس](https://huggingface.co/) –- وب‌سایت اصلی ما –- پلت‌فرمی مرکزی است که به همه امکان مشارکت، کشف و استفاده از جدیدترین و بهترین مدل‌ها و دیتاست‌ها را می‌دهد. این پایگاه، طیف گسترده‌ای از مدل‌ها را در خود جای داده، که بالغ بر ۱۰ هزار مورد از آن‌ها در دسترس عموم قرار دارد. در این فصل بر روی مدل‌ها متمرکز شده و در فصل ۵ نیز نگاهی به دیتاست‌ها خواهیم داشت.
+
+مدل‌های موجود در هاب، محدود به ترنسفورمرهای هاگینگ‌فیس یا حتی پردازش زبان طبیعی نیستند. از جمله آنها می‌توان مدل‌هایی از قبیل [Flair](https://github.com/flairNLP/flair) و [AllenNLP](https://github.com/allenai/allennlp) برای پردازش زبان طبیعی، [Asteroid](https://github.com/asteroid-team/asteroid) و [pyannote](https://github.com/pyannote/pyannote-audio) برای پردازش گفتار، و [timm](https://github.com/rwightman/pytorch-image-models) را برای پردازش تصویر برشمرد.
+
+هر یک از این مدل‌ها به عنوان مخزنی از Git میزبانی می‌شوند که این امر، امکان نسخه‌بندی[^1] و قابلیت بازتولید[^2] را فراهم می‌نماید. به‌اشتراک‌گذاشتن مدل در پایگاه، به معنای باز کردن آن به روی جامعه کاربران و در دسترس قرار دادن آن برای افرادی است که می‌خواهند به راحتی از آن استفاده کنند؛ در نتیجه، ضرورت اینکه افراد مدل را خودشان به تنهایی آموزش دهند، برطرف شده و هم‌رسانی و استفاده از آن تسهیل می‌شود.
```

Comment:
> No one will ever say مخزن for repository in real life. Can we transliterate? Also I think maybe the preposition here should be در instead of از?

Yes, you're spot on for proposition. On the repository thing, please check this out and maybe we'll decide afterwards:
http://4xmen.ir/ترجمه-صحیح-تمامی-اصطلاحات-گیت-git-به-زبان/

Reply:
I just checked the link, I think the repeated use of "Parsi" instead of "Farsi" as the endonym for Persian, disqualifies anything else that site has to say on translation. But then I did a google search and saw how there are a ton of articles referring to makhzan-e-geet. So, I agree let's go with makhzan.

Reply:
> Please add entries for versioning and reproducibility to the G. I also don't see the need for footnotes here. Also for reproducibility, can we go with "tekrar-paziri"? "baz-tolid" I think is not self-explanatory.

Sure, I'll add them. I totally agree with you on "reproducibility" translation, "تکرارپذیری" feels better; Probably even better in this form: قابلیت تکرارپذیری
On the footnotes matter: this is a pretty common practice that masters / PhD students follow in their Persian Thesis writings. I personally believe it helps clarify things, rather than being an overkill. It's your call, just let me know.

Reply:
> I'm thinking we should transliterate Git, bc it's a word the Persian-speaking industry uses everyday, but the Persian transliteration will clash with Gate, what do you think? Maybe we could make an exception for this one and use the English. We should generally avoid that tho.
> 
> Second thought, I think we should go ahead and transliterate, people reading this have software context in mind. No confusion with equivalent of gate.

The "gate" resemblance is exactly what I thought and made me keep it as "Git". I did some googling now, and looks like both "مخزن" and "گیت" are common as version tracking terms, especially when used together. Check out the attached screenshot.
![git_repository](https://user-images.githubusercontent.com/24595790/164556063-5d56c7b9-bcf3-48ef-aa86-23f5a92a13eb.png)

Reply:
> > Please add entries for versioning and reproducibility to the G. I also don't see the need for footnotes here. Also for reproducibility, can we go with "tekrar-paziri"? "baz-tolid" I think is not self-explanatory.
> 
> 
> 
> Sure, I'll add them. I totally agree with you on "reproducibility" translation, "تکرارپذیری" feels better; Probably even better in this form: قابلیت تکرارپذیری
> 
> On the footnotes matter: this is a pretty common practice that masters / PhD students follow in their Persian Thesis writings. I personally believe it helps clarify things, rather than being an overkill. It's your call, just let me know.

You know what I think the problem with footnotes?

1. If you're a newbie, and you see a word like tekrar-paziri with a footnote to an unfamiliar and in this case big English word like REPRODUCIBILITY, you get the idea that this is something you won't understand and there is more to it than the simple idea at its core which Persian can sufficiently express.

2. It's like the author did not believe Persian was enough of a language to express this simple concept fully without needing to resort to linking into English lexicon.

3. Some of these pages are really long, like mine in chapter 2. I get how people would do that in papers and books but here it might as well be an endnote bc it is totally obscured from view when encountering the word and prompts scrolling out of your focus zone only to come across a practically useless English equivalent, which we already have in the glossary--the endnote of all endnotes.

4. There might be cases where the English footnote is of actual use, but VERSIONING and REPRODUCIBILITY hardly seem like it. Like, when transliterating the name of a Person it makes total sense to have footnotes with the English spelling, bc we won't have that name in the glossary and that is information that is lost if we don't have a footnote.